src/posix/platform/netif.cpp

```diff
@@ -1080,6 +1080,45 @@ static otError tryProcessIcmp6RaMessage(otInstance *aInstance, const uint8_t *da
 }
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
 
+#ifdef __linux__
+/**
+ * Returns whether the address is a required anycast address (RFC2373, 2.6.1).
+ *
+ */
+static bool isRequiredAnycast(const uint8_t *aAddress, uint8_t aPrefixLength)
+{
+    bool    isRequiredAnycast = true;
```

Comment:
We can simplify a bit by initializing to `false` and setting `isRequiredAnycast = true` just before the `exit` label below.

Reply:
Thanks! Good catch lol